### PR TITLE
2.12.2

### PR DESCRIPTION
**Pull request by eatyourgreens**
Security update: patches the JSON-API client.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -285,7 +293,6 @@
 
 **Merged pull requests:**
 
-- Alternate fix for `env` variable in lib/config.js [\#23](https://github.com/zooniverse/panoptes-javascript-client/pull/23) ([shaunanoordin](https://github.com/shaunanoordin))
```
Comment (srallen) [inline CHANGELOG.md]: Why was this removed?

Comment (eatyourgreens) [reply CHANGELOG.md]: I'm not sure. I'm generating new versions of the changelog with a Ruby script that compiles the changes from the github API.
